features/0183-revocation-notification/README.md

```diff
@@ -42,19 +42,25 @@ The `revoke` message sent by the `issuer` to the `holder` is as follows:
 {
   "@type": "https://didcomm.org/revocation_notification/1.0/revoke",
   "@id": "<uuid-revocation-notification>",
-  "credential_id": "<uuid-credential>",
+  "~please_ack": ["RECEIPT","OUTCOME"],
```

Comment:
I updated the please ack protocol in #644. As it stands now the syntax is as follows:
```suggestion
  "~please_ack": {
  	"on": "RECEIPT"
  }
```